app/src/main/java/com/keystone/cold/Utilities.java

```diff
@@ -56,6 +56,7 @@
     private static final String ATTACK_DETECTED = "attack_detected";
     private static final String INPUT_SETTINGS_CLEARED = "input_settings_cleared";
     public static final String NET_MDOE = "network_mode";
+    public static final String CASA_SET_UP_FROM_LEGACY = "casa_set_up_from_legacy";
```

Comment:
What's the name of CASA_SET_UP_FROM_LEGACY stands for

Reply:
The number of times the casa guide page appears